docs/process-services/process-list.component.md

```diff
@@ -52,11 +52,12 @@ when the process list is empty:
 | Name | Type | Default value | Description |
 | ---- | ---- | ------------- | ----------- |
 | appId | `number` |  | The id of the app. |
-| data | `DataTableAdapter` |  | Data source to define the datatable. |
+| data | [`DataTableAdapter`](../../lib/core/datatable/data/datatable-adapter.ts) |  | Data source to define the datatable. |
 | multiselect | `boolean` | false | Toggles multiple row selection, which renders checkboxes at the beginning of each row |
 | page | `number` | 0 | The page number of the processes to fetch. |
+| presetColumn | `string` |  | Custom preset column schema in JSON format. |
 | processDefinitionId | `string` |  | The Definition Id of the process. |
-| processInstanceId | `number` |  |  | The id of the process instance. |
+| processInstanceId | `number \| string` |  | The id of the process instance. |
```

Comment:
| processInstanceId | `number \| string` |  | The id of the process instance. |
 something strange here

Reply:
👍